commitlint.config.js

```diff
@@ -1 +1,7 @@
-export default { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+    ignores: [(commit) => {
+      return commit.includes('Signed-off-by: dependabot[bot]');
```

Comment:
I don't think this is the right fix. The actual error is:

`Signed-off-by: dependabot[bot] <support@github.com> ✖ body's lines must not be longer than 100 characters [body-max-line-length]` (see [action log](https://github.com/rancher-sandbox/sbombastic/actions/runs/15823715750/job/44598631218#step:7:24))

So, we might want to set the [body-max-line-length](https://commitlint.js.org/reference/rules.html#body-max-length) rule to `Infinity` instead.